man/systemd-repart.xml

```diff
@@ -89,34 +89,34 @@
       <listitem><para>Taking the size constraints and weights declared in the configuration files into
       account, all partitions that shall be created are now allocated to the disk, taking up all free space,
-      always respecting the size and padding requests. Similar, existing partitions that are determined to
-      grow are grown. New partitions are always appended to the end of the existing partition table, taking
-      the first partition table slot whose index is greater than the indexes of all existing
-      partitions. Partition table slots are never reordered and thus partition numbers are ensured to remain
-      stable. Note that this allocation happens in RAM only, the partition table on disk is not updated
-      yet.</para></listitem>
+      always respecting the size and padding requests. Similarly, existing partitions that should be grown
+      are grown. New partitions are always appended to the end of the partition table, taking the first
+      partition table slot whose index is greater than the indexes of all existing partitions. Partition
+      table slots are never reordered and thus partition numbers are ensured to remain stable. Note that this
+      allocation happens in memory only, the partition table on disk is not updated yet.</para></listitem>
 
       <listitem><para>All existing partitions for which configuration files exist and which currently have no
       GPT partition label set will be assigned a label, either explicitly configured in the configuration or
-      (if that's missing) derived automatically from the partition type. The same is done for all partitions
-      that are newly created. These assignments are done in RAM only, too, the disk is not updated
-      yet.</para></listitem>
+      — if that's missing — derived automatically from the partition type. The same is done for all
+      partitions that are newly created. These assignments are done in memory only, too, the disk is not
+      updated yet.</para></listitem>
 
       <listitem><para>Similarly, all existing partitions for which configuration files exist and which
       currently have an all-zero identifying UUID will be assigned a new UUID. This UUID is cryptographically
       hashed from a common seed value together with the partition type UUID (and a counter in case multiple
       partitions of the same type are defined), see below. The same is done for all partitions that are
-      created anew. These assignments are done in RAM only, too, the disk is not updated
-      yet.</para></listitem>
+      created anew. These assignments are done in memory only, too, the disk is not updated yet.
+      </para></listitem>
 
       <listitem><para>Similarly, if the disk's volume UUID is all zeroes it is also initialized, also
-      cryptographically hashed from the same common seed value. Also, in RAM only, too.</para></listitem>
+      cryptographically hashed from the same common seed value. This is done in memroy only too.
```

Comment:
typo

Reply:
Your machine doesn't have any memroy? You should upgrade. It makes things sooo much better.